project/pom.xml

```diff
@@ -16,6 +16,12 @@
     <failOnMissingWebXml>false</failOnMissingWebXml>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <version>2.10.0</version>
+        <scope>test</scope>
+    </dependency>
```

Comment:
Maybe you can remove this because I see you are using the spring framework MockHttpServletResponse/Request in your tests.

Reply:
needed for `    new SpannerClient().contextInitialized(new ServletContextEvent(mockServletContext));`